### DISCUSSION TITLE
How i can glow a Item

### DISCUSSION THREAD

**Discussion by FlxwDNS** [Minestom/Minestom#1463] [Help] [answered]
Hello, how can i glow this item?

`player.getInventory().setItemStack(2, ItemStack.of(Material.NAME_TAG).withDisplayName(Component.text("§7AutoNick")));`

**Comment by GreatWyrm** [comment, marked as answer]
I assume with glow you mean the enchantment glint.

And that can be accomplished by adding on any sort of enchantment or curse, like so

```java
ItemStack.of(Material.NAME_TAG).withDisplayName(Component.text("AutoNick", NamedTextColor.GRAY)).withMeta(meta -> {
            meta.hideFlag(ItemHideFlag.HIDE_ENCHANTS);
            meta.enchantment(Enchantment.VANISHING_CURSE, (short) 1);
        });
```

Additionally, I would recommend you familiarize yourself with how adventure components work, rather than using color codes and the § symbol.
